test/e2e/common.go

```diff
@@ -484,45 +503,41 @@ func (t *Tester) CheckHubReady() error {
 	// make sure open-cluster-management-hub namespace is created
 	if _, err := t.KubeClient.CoreV1().Namespaces().
 		Get(context.TODO(), t.clusterManagerNamespace, metav1.GetOptions{}); err != nil {
-		return err
+		return fmt.Errorf("failed to get namespace %s: %v", t.clusterManagerNamespace, err)
 	}
 
 	// make sure hub deployments are created
+	// registration-controller
 	if _, err := t.KubeClient.AppsV1().Deployments(t.clusterManagerNamespace).
 		Get(context.TODO(), t.hubRegistrationDeployment, metav1.GetOptions{}); err != nil {
+		return fmt.Errorf("failed to get deployment %s in %s: %v", t.hubRegistrationDeployment, t.clusterManagerNamespace, err)
+	}
+	// registration-webhook
+	registrationWebhookDeployment, err := t.KubeClient.AppsV1().Deployments(t.clusterManagerNamespace).
+		Get(context.TODO(), t.hubRegistrationWebhookDeployment, metav1.GetOptions{})
+	if err != nil {
 		return err
 	}
-	gomega.Eventually(func() error {
```

Comment:
Remove `Eventually` inside of the `CheckHubReady` function, since the function already used with `Eventually` outside.